system_launch/config/system_monitor.yaml

```diff
@@ -0,0 +1,36 @@
+cpu_monitor:
```

Comment:
Fix yaml style.
https://github.com/tier4/AutowareArchitectureProposal.iv/blob/ros2-v0.8.0-beta/system/system_monitor/config/system_monitor.yaml

Reply:
Thanks! I fixed.
I didn't import `num_disks: 2` since the original file doesn't have one.
I will add a line if it is necessary.

Reply:
num_disks was added in ros2 porting.
https://github.com/tier4/AutowareArchitectureProposal.iv/blob/ddd9f8a66b4191740fa9269a962b06ac59f2dcbe/system/system_monitor/src/hdd_monitor/hdd_monitor.cpp#L242
I think it's necessary.

Reply:
Thanks! I fixed and pushed commit.